### ISSUE TITLE
Formatting Enhancement for test output

### ISSUE THREAD

**Issue by gzoller** [scalameta/munit#50]
This is a minor thing, but IMO it would make test output look nicer if the details were indented.
Today:
```
co.blocke.dotty_reflection.ScalaTasty:
+ reflect basic Tasty class with union 1.714s
+ create basic Tasty class 0.002s
co.blocke.dotty_reflection.JavaNonTasty:
+ reflect basic with capture 0.078s
+ create Java object 0.004s
+ Verify Java primitives 0.052s
```
Proposed:
```
co.blocke.dotty_reflection.ScalaTasty:
   + reflect basic Tasty class with union 1.714s
   + create basic Tasty class 0.002s
co.blocke.dotty_reflection.JavaNonTasty:
   + reflect basic with capture 0.078s
   + create Java object 0.004s
   + Verify Java primitives 0.052s
```
This would let your eye catch the suite-level easier.

**Comment by olafurpg** [comment]
Thank you for the suggestion! I agree it looks a lot nicer with indentation. I opened https://github.com/scalameta/munit/pull/54 adding two space indent for successful test case reports.

<img width="213" alt="Screenshot 2020-02-16 at 16 39 30" src="https://user-images.githubusercontent.com/1408093/74608907-0334ae00-50dd-11ea-8278-9590a286d968.png">

**Comment by olafurpg** [comment]
Released as v0.5.2 https://github.com/scalameta/munit/releases/tag/v0.5.2
